lib/house.rb

```diff
@@ -0,0 +1,41 @@
+class House
+  attr_reader :price,
+              :address,
+              :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  # def price
+  #   @price.slice(0).to_i
+  # end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    return true if @price.to_i > 500000
```

Comment:
You can ditch the `if else` statement here since `@price.to_i > 500000` will evaluate to a boolean returning either true or false